pandas/_libs/algos_rank_helper.pxi.in

```diff
@@ -27,7 +27,7 @@ dtypes = [('object', 'object', 'Infinity()', 'NegInfinity()'),
 {{if dtype == 'object'}}
 
 
-def rank_1d_{{dtype}}(object in_arr, bint retry=1, ties_method='average',
```

Comment:
very odd that we had this arg here in the first place

Reply:
> lgtm. small changes. can you add note in the bug fix numeric section. if you think its worth a sub-section ok with that as well (as this is a numerically impactful change and should be highlited a bit).

Btw, where should I put the note ? Which file ?

Reply:
v0.22.0.txt, add a small sub-section in other enhancements with a mini-example.